src/core/config.py

```diff
@@ -13,6 +13,7 @@ class Config:
 
     _raw_language: str = os.environ.get('HUMAN_LANGUAGE', 'en')
     HUMAN_LANGUAGE = LanguageValidator.validate_language(_raw_language)
```

Comment:
Chỗ này cần thêm xử lý lỗi nếu biến môi trường `PRIMARY_MODEL` không hợp lệ. Ví dụ, nếu giá trị không nằm trong danh sách các model được hỗ trợ, cần ném ra exception hoặc đặt giá trị mặc định an toàn hơn.  Cần thêm kiểm tra để đảm bảo tính toàn vẹn của ứng dụng.